azkaban-exec-server/src/main/java/azkaban/container/FlowContainer.java

```diff
@@ -67,51 +60,65 @@
 import java.nio.file.Paths;
 import java.security.KeyStore;
 import java.util.HashMap;
+import java.util.List;
 import java.util.Map;
+import java.util.Optional;
 import java.util.concurrent.ExecutionException;
 import java.util.concurrent.ExecutorService;
 import java.util.concurrent.Executors;
 import java.util.concurrent.Future;
 import java.util.zip.ZipFile;
-import javax.sql.DataSource;
-import org.apache.commons.dbutils.QueryRunner;
-import org.apache.log4j.Logger;
+import org.apache.commons.lang.exception.ExceptionUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
 /**
- *  This class is responsible for launching a flow in a container.
+ *  This class is the entrypoint for launching a flow execution in a container.
+ *  It sets up the Azkaban Properties, the DAO, injects all the required classes, sets up the
+ *  execution directory along with the project files, creates the FlowRunner and submits it to
+ *  the executor service for execution.
+ *  It assumes that their is a certain directory structure consisting of all the dependencies.
+ *  The dependencies such as Hadoop, Hive, Pig, and other libraries.
  */
 public class FlowContainer {
 
   private static final String PROJECT_DIR = "project";
-  private static final String JOBTYPE_DIR = "jobtypes";
+  private static final String JOBTYPE_DIR = "plugins/jobtypes";
   private static final String CONF_ARG = "-conf";
   private static final String CONF_DIR = "conf";
+  private static final String FLOW_NUM_JOB_THREADS = "flow.num.job.threads";
```

Comment:
Can you please rename this variable as well as property? 
Is it number of thread per job? If so, max.job.thread.count or something?

Reply:
This is again a legacy code but I can change it. It is number of threads per job. However, it is not a max.